app/jobs/comments/bust_cache_job.rb

```diff
@@ -0,0 +1,11 @@
+module Comments
+  class BustCacheJob < ApplicationJob
+    queue_as :comments_bust_cache
+
+    def perform(comment_id, cache_buster = CacheBuster.new)
+      comment = Comment.find_by(id: comment_id)
```

Comment:
A guard clause is needed after the `find_by`, in case the comment doesn't exist (e.g was deleted).